src/script/components/list/ConversationListCell.tsx

```diff
@@ -35,17 +35,22 @@ import Avatar from 'Components/Avatar';
 import GroupAvatar from 'Components/avatar/GroupAvatar';
 import AvailabilityState from 'Components/AvailabilityState';
 import Icon from 'Components/Icon';
-import {KEY} from 'Util/KeyboardUtil';
-import {setContextMenuPosition} from 'Util/util';
+import {isKey, isOneOfKeys, KEY} from 'Util/KeyboardUtil';
 
 export interface ConversationListCellProps {
   conversation: Conversation;
   dataUieName: string;
   isSelected?: (conversation: Conversation) => boolean;
   onClick: React.MouseEventHandler<Element>;
   onJoinCall: (conversation: Conversation, mediaType: MediaType) => void;
-  rightClick: (conversation: Conversation, event: MouseEvent) => void;
+  rightClick: (conversation: Conversation, event: MouseEvent | React.MouseEvent<Element, MouseEvent>) => void;
   showJoinButton: boolean;
+  index: number;
+  focus: boolean;
+  showFocus: boolean;
```

Comment:
Can you document, in the code, what is the difference between `focus` and `showFocus` it's not quite clear. 

you can use `jsdoc` style comment so that we get the info on our IDE directly :) 

```suggestion
  /** explain here what the `focus` property does */
  focus: boolean;
  
  /** explain here what the `showFocus` property does */
  showFocus: boolean;
```